osu.Game/Rulesets/UI/Scrolling/ScrollingHitObjectContainer.cs

```diff
@@ -81,8 +81,10 @@ public double TimeAtPosition(float localPosition, double currentTime)
         /// </remarks>
         public double TimeAtScreenSpacePosition(Vector2 screenSpacePosition)
         {
-            Vector2 localPosition = ToLocalSpace(screenSpacePosition);
-            return TimeAtPosition(scrollingAxis == Direction.Horizontal ? localPosition.X : localPosition.Y, Time.Current);
+            Vector2 pos = ToLocalSpace(screenSpacePosition);
+            float localPosition = scrollingAxis == Direction.Horizontal ? pos.X : pos.Y;
+            localPosition -= axisInverted ? scrollLength : 0;
```

Comment:
I guess this transformation seems fine math-wise, but I'm having trouble on how to interpret this value. It looks like it's the relative position of the object across the scrolling axis from the near edge of the scroll... but with the sign in screen coordinates, not the scroll coordinates...?

Might be a good idea to explain that in the xmldoc of `TimeAtPosition` or something...

Reply:
I found that it is hard to explain the thing without mentioning the implied coordinate system (anchoring bottom for down scroll etc.).
But I don't more changes to be included in PR.